packages/graphql-hooks-ssr/README.md

```diff
@@ -27,7 +27,8 @@ app.get('/', async (req, reply) => {
   const client = new GraphQLClient({
     url: 'https://domain.com/graphql',
     cache: memCache() // NOTE: a cache is required for SSR,
-    fetch
+    fetch,
+    ssrMode: true
```

Comment:
This isn't required when using `graphql-hooks-ssr`, see https://github.com/nearform/graphql-hooks/blob/master/packages/graphql-hooks-ssr/index.js#L13. Perhaps a comment here to say that `graphql-hooks-ssr` set the flag will make it clearer.

Reply:
Ok, will fix pull request